### PR TITLE
[user account] fix can't show email errors

### PR DESCRIPTION
**Pull request by kongtiaowang**
## Brief summary of changes
fix the variable name
#### Testing instructions (if applicable)
input an invalid email to create or edit a user, then the error shows on the page.
#### Link(s) to related issue(s)
https://github.com/aces/Loris/issues/7746

Screen Shot 2021-11-05 at 11 44 09 AM
)

### CODE REVIEW

File: modules/user_accounts/templates/form_edit_user.tpl [hunk 1]
```diff
@@ -266,9 +265,11 @@
     	</div>
     </div>
     {/if}
-
-    {if $form.errors.Email|default}
+    {if $form.errors.Email_Group|default}
```
Comment (ridz1208) [inline modules/user_accounts/templates/form_edit_user.tpl:268]: I don't think this is the correct solution, you are just undoing work from #7274 I think what should be done instead is to replace email_group in the edit_user class by "email" in the validate function.

**Review by ridz1208 — CHANGES_REQUESTED** [review]

**Comment by ridz1208** [comment]
@kongtiaowang have you checked that other fields in that same form haven't changed names as well ?

Read the issue (#7746) carefully there are more details there

**Comment by kongtiaowang** [comment]
@ridz, could you review again?

**Review by ridz1208 — APPROVED** [review]
Code looks good

**Comment by ridz1208** [comment]
@racostas could you test please ?

**Comment by racostas** [comment]
LGTM

... a related issue:

When a user request an account by itself (not created by the admin) the validations in the email field are not triggered. We should do this consistent.

image

image

**Comment by ridz1208** [comment]
@racostas please make a separate issue for what you have found

**Pull request merged**